ts/third_party/cloud-debug-nodejs/src/agent/io/scanner.ts

```diff
@@ -144,6 +144,8 @@ function computeStats(
         }
         statistics[filename] = fileStats;
       } catch (err) {
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
This is a git subtree, but it is in a broken state because of a branch rename upstream. The previous team also edited this file directly for gts/ts upgrades. I tried pulling in a new version but it had some breaking changes :(

Reply:
So upgrading to a newer version would be handled in a separate release ?